commitlint.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
       2,
       'always',
       (parsed) => {
-        const ticketNumberFormat = /^cvsb-\d+$/;
+        const ticketNumberFormat = /^cb2-\d+$/;
```

Comment:
Do we want to update this in the template?

Reply:
I would say so, its our new ticket number strategy after all